portfoliostyle.css

```diff
@@ -0,0 +1,438 @@
+
+
+* {
+	box-sizing: border-box;
+	font-family: 'Nunito', sans-serif;
+}
+
+section, footer div {
+	width: 65%;
+	margin: 0 auto;
+}
+
+body {
+	background-color: #D3D3D3;
+	background-image: linear-gradient(#808080, #FFFFFF);
+	background-position: top center;
+}
+
+/*most element styles*/
+
+nav {
+	position: fixed;
+	top: 1vh;
+	right: 1vw;
+	background-color: #808080;
+	border: 3px solid #FFFFFF;
+	opacity: 0.7;
+	border-radius: 30%;
+	padding: 10px 25px;
+	margin: 20px;
+	display: flex;
+	flex-direction: column;
+}
+
+nav:hover {
+	opacity: 0.7;
+	background-color: #000000;
+}
+
+nav a {
+	text-decoration: none;
+}
+
+nav a:hover {
+	color: #ADD8E6;
+}
+
+header {
+	height: 100vh;
+	background-image: url('Nova1.jpeg');
+	background-repeat: no-repeat;
+	background-attachment: fixed;
+	background-size: cover;
+	color: #FFFFFF;
+}
+
+h1 {
+	font-size: 100px;
+	margin: 0;
+	padding: 100px;
+}
+
+h2, h3 {
+	text-align: center;
+	background-color: #FFFFFF;
+}
+
+h2 {
+	margin-top: 10vh;
+	font-size: 40px;
+	border-radius: 30px;
+	padding: 20px 0;
+}
+
+h3 {
+	color: #000000;
+	border-radius: 15px;
+	padding: 5px 0;
+}
+
+p a {
+	color: #FFFFFF;
```

Comment:
Lots of elements with color: FFFFFF; which you could perhaps combine into one and cut down the number of lines of code:)